src/main/java/seedu/address/logic/commands/ViewTasksCommand.java

```diff
@@ -0,0 +1,24 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_DO_NOT_SHOW_ALL_PERSONS;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_TASKS;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+
+
+public class ViewTasksCommand extends Command {
+
+    public static final String COMMAND_WORD = "viewTasks";
+
+    public static final String MESSAGE_SUCCESS = "Listed all Tasks";
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+        model.updateFilteredPersonList(PREDICATE_DO_NOT_SHOW_ALL_PERSONS);
```

Comment:
Are we supposed to update the person list?

Reply:
ohhh okay can i will make the changes